Trellix/trellix-edr/tests/detection_event.json

```
@@ -0,0 +1,35 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "trellix-edr",
+        "dialect_uuid": "954a6488-6394-4385-8427-621541e881d5"
+      }
+    },
+    "message": "{\"type\":\"detections\",\"id\":\"652404\",\"threatId\":\"182612\",\"attributes\":{\"traceId\":\"9a718cc6-d8f6-46da-b3cc-fc4dbbd60151\",\"firstDetected\":\"2023-08-27T05:34:29Z\",\"severity\":\"s4\",\"rank\":270,\"tags\":[\"@ATA.PrivilegeEscalation\",\"@ATA.Persistence\",\"@ATE.T1546.012\",\"@MSI._reg_ep0130_imageexecution_high\",\"@ATA.DefenseEvasion\",\"@ATE.T1112\"],\"host\":{\"os\":{},\"netInterfaces\":[],\"traceExtendedVisibility\":0,\"hostOs\":\"\",\"aGuid\":\"6D0A37A8-B5B7-4414-9444-A2B17721642B\"},\"sha256\":\"6E2918727CBB836F4D8E3404BDE9AEAF5D4DED5DD1F6916AAD3F3B956E6D8A17\"}}"
+  },
+  "expected": {
+    "message": "{\"type\":\"detections\",\"id\":\"652404\",\"threatId\":\"182612\",\"attributes\":{\"traceId\":\"9a718cc6-d8f6-46da-b3cc-fc4dbbd60151\",\"firstDetected\":\"2023-08-27T05:34:29Z\",\"severity\":\"s4\",\"rank\":270,\"tags\":[\"@ATA.PrivilegeEscalation\",\"@ATA.Persistence\",\"@ATE.T1546.012\",\"@MSI._reg_ep0130_imageexecution_high\",\"@ATA.DefenseEvasion\",\"@ATE.T1112\"],\"host\":{\"os\":{},\"netInterfaces\":[],\"traceExtendedVisibility\":0,\"hostOs\":\"\",\"aGuid\":\"6D0A37A8-B5B7-4414-9444-A2B17721642B\"},\"sha256\":\"6E2918727CBB836F4D8E3404BDE9AEAF5D4DED5DD1F6916AAD3F3B956E6D8A17\"}}",
+    "event": {
+      "kind": "event",
+      "start": "2023-08-27T05:34:29Z"
```

I suggest adding `event.category` and `event.type`
```suggestion
      "start": "2023-08-27T05:34:29Z",
      "category": ["intrusion_detection"],
      "type": ["info"]
```